packages/documentsearch/src/searchview.tsx

```diff
@@ -322,10 +344,18 @@ interface ISearchOverlayProps {
    * Whether the document is read-only or not.
    */
   isReadOnly: boolean;
+  /**
+   * Whether to preserve case when replacing.
+   */
+  preserveCase: boolean;
   /**
    * Whether or not the replace entry row is visible
    */
   replaceEntryVisible: boolean;
+  /**
+   * Support for replace options
+   */
+  replaceOptionsSupport: IReplaceOptionsSupport | undefined;
```

Comment:
Should we use the more common pattern:

```suggestion
  replaceOptionsSupport?: IReplaceOptionsSupport;
```

Should we use the smaller name `replaceOptions`?